service/src/main/kotlin/io/provenance/api/frameworks/web/misc/ServerRequestExt.kt

```diff
@@ -17,12 +20,23 @@ suspend inline fun <reified T> ServerRequest.requireBody(): T =
         requireNotNull(awaitBodyOrNull()) { "Failed to parse request body of type ${T::class}" }
     }
 
-fun ServerRequest.getUser(): UUID {
+fun ServerRequest.getUser(): UserID {
```

Comment:
root of all changes

Question: is this a good name or is EntityID (that can be a UUID or address) more accurate? Went with user since it matched the function name

Reply:
Entity is more accurate because our "users" could be individuals or keys that represent whole organizations. Agreed that we should keep the naming in line so renaming to `getEntity` at the same time would be wise.

Reply:
updated